src/PKSim.Core/Commands/UpdateParameterValueOriginCommand.cs

```diff
@@ -1,46 +1,39 @@
-﻿using OSPSuite.Core.Commands;
+﻿using OSPSuite.Core.Commands.Core;
 using OSPSuite.Core.Domain;
-using OSPSuite.Utility.Extensions;
-using PKSim.Core.Model;
+using Command = OSPSuite.Assets.Command;
 
 namespace PKSim.Core.Commands
 {
-   public class UpdateParameterValueOriginCommand : PKSimMacroCommand
+   public class UpdateParameterValueOriginCommand : EditParameterCommand
```

Comment:
Need to write my own command and cannot use the one in core. Just too much stuff to do to syncrhonize building block otherwise. SImply not worth it